requirements.txt

```diff
@@ -1,8 +1,8 @@
 flask>=1.0.2
 click>=6.7
 markdown>=2.6.11
+markdown-full-yaml-metadata>=0.0.2
```

Comment:
This extension is no more used. Now we use the Meta Extension (its the official); this is a mistake. Please can you delete it? thanks.

Reply:
Consider it done